pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -346,6 +352,12 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 					Path: reservation.GetPrHelperSocketDir(),
 					Type: &directoryOrCreate,
 				},
+			}}, corev1.Volume{
```

Comment:
Maybe small unit test to ensure we don't drop it in the future?

Reply:
We have to test framework for this. Would you mind if I will do it in a follow up PR? The fix is quite urgent

Reply:
Sure! you can do it in this suite https://github.com/kubevirt/kubevirt/blob/882c3a586e3f5ab6cea6f3cc0ae6503ba845cc11/pkg/virt-operator/resource/apply/apps_test.go#L369-L380